python/ray/train/tensorflow/tensorflow_predictor.py

```diff
@@ -121,36 +153,7 @@ def build_model(self):
 
             predictions = predictor.predict(data)
```

Comment:
Can we also comment on what is the return type of predictions?

Reply:
Updated the docstring!

Reply:
regardless of image or tabular etc.

if it is is a single column dataframe, then it will be converted to a single tensor before being inputted to the model.
if it is a multi column dataframe, then it will be converted to a dict of tensors before being inputted to the model.